torch/onnx/_type_utils.py

```diff
@@ -108,36 +113,86 @@ def from_name(
     @classmethod
     @_beartype.beartype
     def from_dtype(cls, dtype: torch.dtype) -> JitScalarType:
-        """Convert a torch dtype to ScalarType."""
+        """Convert a torch dtype to JitScalarType.
+
+        NB: DO NOT USE this API if dtype is comes from a torch._C.Value.type().dtype() call because
```

Comment:
Even though pytorch tends to use `NB` in comments, I suggest we use `Note:` because the docstring will be generated as part of the doc, and sphinx knows how to handle `Note:` better.

```
Note:
    xxxxxx
```

just like Args and Returns, I'd be sure to return and indent

Reply:
done